feature-pack/src/main/resources/content/bin/wsconsume.bat

```diff
@@ -3,6 +3,8 @@
 @if not "%ECHO%" == ""  echo %ECHO%
 @if "%OS%" == "Windows_NT" setlocal
 
+set SERVER_OPTS=%*
```

Comment:
_Maybe_ this needs to be quoted. `set "%SERVER_OPTS%=%*"`. I'm not 100% certain though.